5.37.0/upgradeLegacyRteToCmsFeatureFlag.ts

```diff
@@ -0,0 +1,56 @@
+import { Files } from "../utils";
+import { Project } from "ts-morph";
+import { Context } from "../types";
+import { featureFlagExist } from "../utils/featureFlagExist";
+import { addFeatureFlag } from "../utils/addFeatureFlag";
+
+interface Params {
+    files: Files;
+    project: Project;
+    context: Context;
+}
+
+export const upgradeLegacyRteToCmsFeatureFlag = async (params: Params) => {
+    const { project, files, context } = params;
+
+    context.log.info(
+        "Start migrating feature flag for legacy Rich Text editor support in the Headless CMS app."
```

Comment:
Something I figured out along the way...

Just adding a message at the beginning of the process, like in this case "Adding feature flag for ..." is enough.

Literally just this message in the terminal.

And then only show additional messages if warning/errors happened.

Makes the terminal output cleaner.